src/components/Footer/components/Copyright.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Box, Typography, Link } from "@mui/material";
+
+export default function Copyright() {
+  
+  const date = new Date();
+  const fullDate = `${date.getFullYear()}`;
+
+  return (
+    <Box>
+      <Typography variant="body2" color="text.secondary">
+          Copyright @
+        <Link color="inherit" underline="hover" href="https://softserveinc.com/">
+          {` SoftServe `}
+        </Link> 
+          {fullDate}
+      </Typography>
```

Comment:
use flex to escape adding space like this

Reply:
done. Please check